heka/_common.sls

```diff
@@ -8,6 +8,15 @@ heka_packages:
   file.recurse:
   - source: salt://heka/files/lua
 
+/usr/share/lma_collector/common/extra_fields.lua:
```

Comment:
This would mean adding another directory to `module_dir`. And right now, each service supporting Heka needs to set `module_dir` for each Heka sandbox it defines. I'd suggest that we fix this problem before we add new directory to `module_dir`.